14_functions_lab/03.py

```diff
@@ -1,5 +1,23 @@
-"""
-Write a function that calculates the sum
-of the 10th digit from all arguments passed to it
-"""
-
+"""
+function sums the tenth numbers
+"""
+from types import *
+
+
+
+def func(*args):
+    res = 0
+    for x in args:
+        if (type(x) is int) and (int(x) >= 10): res += x / 10 % 10

```

Comment:
Notice how the main content of the function (to add the tens digit) is indented to the far right. This makes code harder to understand.

Can you modify the code so that part is in the center?